src/clj/game/cards-assets.clj

```diff
@@ -238,28 +238,28 @@
    {:in-play [:hand-size-modification 4]}
 
    "Daily Business Show"
-   {:events {:corp-draw
+   {:events {:pre-corp-draw
              {:msg "draw additional cards"
               :once :per-turn
-              :once-key :daily-business-show
-              :req (req (first-event state side :corp-draw))
-              :effect (req
-                        (let [dbs (count (filter #(and (rezzed? %) (= (:title %) "Daily Business Show"))
-                                                  (all-installed state :corp)))
-                              newcards (take dbs (:deck corp))
-                              drawn (concat newcards (take-last target (:hand corp)))]
-                          (if (not= (count newcards) dbs)
-                            ; couldn't draw from R&D, so corp is decked
-                            (win-decked state)
-                            (do (doseq [c newcards] (move state side c :hand))
-                                (resolve-ability
-                                  state side
-                                  {:prompt (str "Choose " dbs " card" (if (> dbs 1) "s" "") " to add to the bottom of R&D")
-                                   :choices {:max dbs
-                                             :req #(and (in-hand? %)
-                                                        (some (fn [c] (= (:cid c) (:cid %))) drawn))}
-                                   :msg (msg "add " dbs " card" (if (> dbs 1) "s" "") " to bottom of R&D")
-                                   :effect (req (doseq [c targets] (move state side c :deck)))} card targets)))))}}}
+              :once-key :daily-business-show-draw-bonus
+              :req (req (first-event state side :pre-corp-draw))
+              :effect (req (let [dbs (count (filter #(and (rezzed? %) (= (:title %) "Daily Business Show"))
```

Comment:
I know this comparison with the card's title is legacy, but perhaps we should consider using the `:code` of the card instead of the `:title`, which will be a faster string comparison. The `:code` for DBS is `"06086"`, from the URL to the card on NetrunnerDB: https://netrunnerdb.com/en/card/06086